src/components/EditorSideBar/Folder.tsx

```diff
@@ -7,17 +7,22 @@ import { mockTileData } from "../MockData/MockTileData";
 import { useStoreSelector } from "@/store/hooks";
 import { changeWholeCourtColor } from "@/store/reducer/tileSlice";
 import { ActionCreators } from "redux-undo";
+import FolderListItem from "../FolderList/FolderListItems";
 
-const Folder: React.FC = () => {
+const Folder = () => {
   const dispatch = useDispatch();
   const { designsData, activeCourt } = useStoreSelector((state) => state.courtSpecData);
+  console.log(designsData);
+  console.log("below is active");
+  console.log(activeCourt);
   const [activateDesign, setActivateDesign] = useState<string>(activeCourt.courtId);
   const { designTileList } = useStoreSelector((state) => state.designTileList);
 
-  useEffect(() => {
-    if (designsData === undefined) return;
-    setActivateDesign(activeCourt.courtId);
-  }, [designsData]);
+  // useEffect(() => {

```

Comment:
Please remove this function if it is useless :)

Reply:
Thanks Tim for picking this up~ it's removed now.